projects/coordinator/src/main/java/org/batfish/coordinator/ExceptionMapper.java

```diff
@@ -12,11 +12,13 @@ public Response toResponse(Exception exception) {
     if (exception instanceof WebApplicationException) {
       // An exception that already maps directly to a standard HTTP Response, likely thrown
       // somewhere inside Jersey aka if the user typed a bad URI.
-      return ((WebApplicationException) exception).getResponse();
+      return Response.status(((WebApplicationException) exception).getResponse().getStatus())
+          .entity(Throwables.getStackTraceAsString(exception))
```

Comment:
please do not modify the existing response generically like this. For instance, we do not want stack trace for 404 errors.

Find where the exception is being generated and customize the entity there.

Additionally, in the future we may be throwing our own WebApplicationExceptions with good entities, and we don't want to override that to bad entities here.